### ISSUE TITLE
Metro dash doesn't hide anime correctly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#366]
```
What steps will reproduce the problem?
1. Select Metro dash
2.
3.

What is the expected output? What do you see instead?
Anime shown to me with restrictions based on my filters. Ie Hiding pg13 anime 
when in family friendly account. Instead the anime shown are shown regardless 
of tags.

What version of the product are you using? On what operating system?
3.1.26 JMM desktop

Please provide any additional information below.
It would be nice to have the ability to scroll vertically instead of 
horizontally.

```

Original issue reported on code.google.com by `hobbysavvy` on 26 Dec 2012 at 2:45

**Comment by GoogleCodeExporter** [comment]
```
Ok seems like it is working in the continue watching section, but not the other 
sections
```

Original comment by `werndly...@gmail.com` on 26 Dec 2012 at 10:59
- Changed state: **Accepted**
- Added labels: **Priority-High**, **Product-JMM**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.28
```

Original comment by `werndly...@gmail.com` on 26 Dec 2012 at 11:41
- Changed state: **Fixed**
